addbookfunction.php

```diff
@@ -0,0 +1,13 @@
+<?php
+require 'functions.php';
+$db = startDb('books');
+
+if (!empty($_POST)) {
+    if (!empty($_POST['bookName']) && !empty($_POST['author']) && !empty($_POST['category']) && !empty($_POST['released'])) {
+        $bookInfo = ['name' => $_POST['bookName'], 'author' => $_POST['author'], 'category' => $_POST['category'], 'released' => $_POST['released']];
```

Comment:
why do you need this line? surely you can just pass `$_POST` as your array into `addBook`